### PR TITLE
build assets with webpack instead of gulp

### PR DESCRIPTION
**Pull request by smildev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ composer install
 Install project :
 ```bash
 $ bin/console app:install
-$ yarn install && yarn run gulp
+$ yarn install && yarn run encore production (or yarn run encore dev-server --hot for developpement)
```
Comment (loic425) [inline README.md]: @smildev Maybe we could add two yarn scripts on package.json:
```
"scripts": {
    "build": "yarn encore production"
    "dev": "yarn encore dev-server --hot"
     ....
  }
```

Comment (gabiudrescu) [reply README.md]: development, not develo**pp**ment

**Comment by loic425** [comment]
@smildev I fixed the typo in the comment written by @gabiudrescu.
I think we can merge this one :)

**Comment by gabiudrescu** [comment]
whoho, this means that Sylius can backport this to their repo and replace gulp with the newer and meaner Webpack & Encore?

**Comment by loic425** [comment]
> whoho, this means that Sylius can backport this to their repo and replace gulp with the newer and meaner Webpack & Encore?

Yes this is possible, we use something pretty similar for a full Sylius project ah Mobizel.
There is already a pull request on Sylius standard repository https://github.com/Sylius/Sylius-Standard/pull/354

**Pull request merged**